server/statistics/topn.go

```diff
@@ -117,59 +106,129 @@ func (tn *TopN) RemoveExpired() {
 }
 
 // Remove deletes the item by given ID and returns it.
-func (tn *TopN) Remove(id uint64) TopNItem {
+func (tn *TopN) Remove(id uint64) (item TopNItem) {
 	tn.rw.Lock()
 	defer tn.rw.Unlock()
-	item := tn.removeItemLocked(id)
+	for _, stn := range tn.topns {
+		item = stn.Remove(id)
+	}
 	_ = tn.ttlLst.Remove(id)
 	tn.maintain()
-	return item
+	return
```

Comment:
It seems no need to return

Reply:
I think return the removed item is a better API. And it is used in the test.